perf/src/perf_libs.rs

```diff
@@ -16,7 +15,7 @@ use {
 
 #[repr(C)]
 pub struct Elems {
-    pub elems: *const Packet,
+    pub elems: *const u8,
     pub num: u32,
```

Comment:
@joncinque 
Seems like `num` field here is the number of `Packet`s, not bytes.
So if `elems` does not have a known type, `num` becomes ambiguous.
e.g. how would the `C` code iterate over `elems`  if their size is unknown.

Reply:
That's correct that `num` refers to the number of elements, and not their size.

If you look at the API that uses `Elems`, however, which is `ed25519_verify_many` and `ed25519_sign_many`, you'll see that we pass `message_size`, which defines the size of each packet. In fact, we currently pass in `size_of::<Packet>() as u32` for that parameter. So the size is not defined by this type here.

You can even see the underlying function calls in `solana-perf-libs` starting at https://github.com/solana-labs/solana-perf-libs/blob/233c4457fd8d425da99cb3db46b2be8eb8b98997/src/cuda-ecc-ed25519/ed25519.h#L35, where they just use a `uint_8 *` underneath.